exercises/practice/reverse-string/src/ReverseString.hs

```diff
@@ -0,0 +1,3 @@
+module ReverseString (reverseString) where
+
+reverseString :: String -> String
```

Comment:
```suggestion
reverseString :: String -> String
reverseString :: str = error "You need to implement this function."
```

Reply:
Weird, the accepted suggestion from Matthijs' review comment seems to have deleted the line instead of updating it 🤔 
Good catch, will fix.